src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/DataController.cs

```diff
@@ -495,5 +507,15 @@ private Instance GetInstance(string instanceId, int instanceOwnerId, out ActionR
 
             return null;
         }
+
+        internal string GetUser(ClaimsPrincipal user)
```

Comment:
As the method gets name of the user the name of the function could be more precise. eg. GetNameOfUser / GetUserName

Reply:
Fixed to GetNameOfUser